### DISCUSSION TITLE
where can I paste it?

### DISCUSSION THREAD

**Discussion by Wangggym** [gitkraken/vscode-gitlens#3551] [Q&A]
![image](https://github.com/user-attachments/assets/c62c120c-5854-494f-8006-52d2551d4569)

**Comment by Wangggym** [comment]
Additional functionality is now available to you in GitLens and the GitKraken DevEx platform. Copy and paste this into GitLens

**Comment by jkelroy** [comment]
Hello @Wangggym, you should have a prompt in VS Code to provide the token. 

![Screenshot 2024-08-30 at 10 43 00 AM](https://github.com/user-attachments/assets/3a5af96e-426d-4aad-8abf-4fad41ff1619)

If you do not see this, it should have been automatically provided and you should be signed in. Are you not seeing your account when looking in GitLens - this can be seen from command/ctrl + shift + P > _GitLens: Focus on GitKraken Account View_?

**Reply by Wangggym** [reply]
@jkelroy Thanks, it works for me. My vs code has worked, The cursor editor has this problem, use your solution, it fixed.

**Reply by harmenjanssen** [reply]
> If you do not see this, it should have been automatically provided and you should be signed in. Are you not seeing your account when looking in GitLens - this can be seen from command/ctrl + shift + P > _GitLens: Focus on GitKraken Account View_?

In Windsurf editor there is no _GitLens: Focus on GitKraken Account View_. 
Also no input field to paste the token into. 

Is this deliberately not available in Windsurf?

**Comment by mbalc** [comment]
this is super confusing, you should add any kind of info on that page - as when someone accidentally closes their prompt for the api key, they're left lost to search the internet what you mean ("where the heck am i supposed to paste that thing?") and find this issue, just like I just did
